ts/models/IdplLoginPhase.ts

```diff
@@ -0,0 +1,10 @@
+export type IdplLoginPhase = {
+  id: string;
+  detail: string;
```

Comment:
if `detail` takes a predefined set of values, declare them in the type and use constant like identifiers, e.g.:

```suggestion
  detail: "START" | "NETWORK_ERROR";
```